base_tier_validation/static/src/js/services/tier_review_service.esm.js

```diff
@@ -0,0 +1,32 @@
+import {registry} from "@web/core/registry";
+
+export class TierReviewService {
+    constructor(env, services) {
+        this.env = env;
+        this.store = services["mail.store"];
+        this.busService = services.bus_service;
+    }
+    setup() {
+        this.busService.subscribe("base.tier.validation/updated", (payload) => {
+            if (payload.review_created) {
+                this.store.tierReviewCounter++;
+            }
+            if (payload.review_deleted) {
+                this.store.tierReviewCounter--;
+            }
+        });
+        this.busService.start();
```

Comment:
Please check this issue; real-time updates are not working. Please review the video where it works fine in V17.